### PR TITLE
Updating to java-jwt 2.0.1

### PR DESCRIPTION
**Pull request by pose**

### CODE REVIEW

File: docs/server-apis/java.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ public class JWTFilter implements Filter {
     @Override
     public void init(FilterConfig filterConfig) throws ServletException {
         jwtVerifier = new JWTVerifier(
-          new Base64(true).decodeBase64("<%= account.clientSecret %>"),
+          "<%= account.clientSecret %>",
```
Comment (mgonto) [inline docs/server-apis/java.md]: Hey,

The new `JWTVerifier` receives a NON base64URL Encoded secret. You have to live the transformer I think.

Comment (pose) [inline docs/server-apis/java.md]: You are right :). Good catch :+1:

**Comment by mgonto** [comment]
Now it's mergeable :+1:  :)

**Comment by pose** [comment]
Thanks :D!

**Pull request merged**